act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
-===================================
-
-An ACT Rule MAY contain information about the background for the development of the rule, or references to relevant reading. The relationship to the relevant reading should be specified. Examples of relevant background references for a rule for a WCAG [[WCAG]] success criterion could be [Understanding documents](https://www.w3.org/WAI/WCAG21/Understanding/), [WCAG Techniques](https://www.w3.org/WAI/WCAG21/Techniques/), or WAI-ARIA [[WAI-ARIA]], CSS [[css-2018]], or HTML [[HTML]] specifications.
-
-
 Rule Accuracy {#accuracy}
 =========================
 
 This section is *non-normative*.
 
-While [test cases](#test-cases) can be used to determine if an ACT Rule was correctly implemented, they do not guarantee that implementations will never produce incorrect results. When writing ACT Rules, it is almost inevitable that edge cases will be overlooked. Technologies are always evolving, and content authors are constantly coming up with new and unexpected ways to use those.
+While [test cases](#test-cases) can be used to determine if an ACT Rule was correctly implemented, they do not guarantee that implementations will never produce incorrect results. When writing ACT Rules, it is almost inevitable that edge cases will be overlooked. Technologies are always evolving, and content authors are constantly coming up with new and unexpected ways to use those. Some examples of causes for inaccuracy are:
+
+- [Assumptions](#assumptions) were made about the test subject that turned out to be untrue
+- Technologies were used in an unusual and difficult to predict manner
+- Technologies have changes, or aspects of the technologies were overlooked
+- The accessibility requirement was not correctly interpreted
 
-There are two types of inaccuracies that can produce incorrect results. Inaccuracies in the **implementation** can be addressed with test cases, but inaccuracies in the **ACT Rule** itself can not. After all, rule inaccuracies come from the rule author being unaware of a particular edge case.
+There are two types of inaccuracies that can produce incorrect results. Inaccuracies in the **implementation** can be addressed with test cases, but inaccuracies in the **ACT Rule** itself cannot. After all, rule inaccuracies come from the rule author being unaware of a particular edge case.
 
 When a test result incorrectly indicates non-conformance to an accessibility requirement, this is known as a false positive. Opposite, when a rule incorrectly indicates conformance, this is a false negative. A percentage of false positives and false negatives can be calculated by comparing it to results from an accessibility audit:
 
-- **False positives**: This is the percentage of [=test subjects=], that were `failed` by the rule, but conform to the [=accessibility requirements=].
+- **False positives:** This is the percentage of [=test subjects=], that were `failed` by the rule, but conform to the [=accessibility requirements=].
```

Comment:
Should it be "test targets" or "test subjects" here (and below)? 

Because now "test targets" are the ones to have a passed/failed outcome - but the requirement still works on a "test subject" level?

Reply:
Change "test subject" to "test target" and "conform" to "satisfy".  Test targets don't conform to WCAG, pages conform to WCAG.  So both these changes are needed.